configure

```diff
@@ -388,22 +361,17 @@ SWISH_LIBPATH="${BASE}/build/release/bin"
 LIBUV_INCLUDE="${BASE}/libuv/include"
 case "${CONFIG_UNAME}" in
   CYGWIN_NT-* | MINGW64_NT-* | MSYS_NT-*)
-    UvLib=libuv/Release
-    ;;
-  Darwin)
     UvLib=libuv/build/Release
     ;;
   *)
-    UvLib=libuv/out/Release/obj.target
+    UvLib=libuv/build
     ;;
 esac
-LIBUV_LIBPATH="${BASE}/${UvLib}"
 if [ "${Windows}" = "yes" ]; then
     SCHEME="$(cygpath -au "${SCHEME}")"
     SWISH_INCLUDE="$(cygpath -aw "${SWISH_INCLUDE}")"
     SWISH_LIBPATH="$(cygpath -aw "${SWISH_LIBPATH}")"
     LIBUV_INCLUDE="$(cygpath -aw "${LIBUV_INCLUDE}")"
-    LIBUV_LIBPATH="$(cygpath -aw "${LIBUV_LIBPATH}")"
```

Comment:
I think this was a little too aggressive deletion. If I have code that needs uv.lib, we need to know where to find it. (LIBUV_LIBPATH)